src/frontend/csrs-portal/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "configure": "ts-node setenv.ts",
-    "start": "yarn run configure -- --environment=dev && ng serve",
+    "start": "yarn run configure -- --environment=dev && ng serve --proxy-config proxy.conf.json",
```

Comment:
this works, but you could also add another script, does everyone have to run with proxy?

Reply:
It was added a new script start-proxy.